src/types/Trigger.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Interface for building a trigger based on time.
+ *
+ * View the [Triggers](/react-native/docs/triggers) documentation to learn more.
+ *
+ * Currently only supported on Android.
+ *
+ * @platform android
+ */
+export interface TimeTrigger {
+  /**
+   * Constant enum value used to identify the trigger type.
+   */
+  type: TriggerType.TIME;
+  /**
+   * The timestamp when the notification should first be shown, in milliseconds since 1970.
+   */
+  timestamp: number;
+
+  /**
+   * How frequently the notification should be repeated.
+   *
+   * If not present, the notification will be displayed once.
+   *
+   * For example, if set to 30, the notificaiton will be displayed every 30 minutes.
+   *
+   * Must be set to a minimum of 15 minutes.
+   *
+   */
+  repeatInterval?: number;
+
+  /**
+   * The unit of time that the `repeatInterval` is measured in.
+   *
+   * For example, if set to `TimeUnit.DAYS` and repeat interval is set to 3, the notificaiton will repeat every 3 days.
```

Comment:
```suggestion
   * For example, if set to `TimeUnit.DAYS` and repeat interval is set to 3, the notification will repeat every 3 days.
```

slightly more useful, but only slightly, a typo spotted